content/en/getting_started/tracing/_index.md

```diff
@@ -20,159 +20,201 @@ further_reading:
 
-If you haven't already, create a [Datadog account][3].
+## Prerequisites
 
-## Datadog Agent
+To complete this guide, you need the following:
 
-Before installing the Datadog Agent, set up a [Vagrant Ubuntu 22.04 virtual machine][4] using the following commands. For more information about Vagrant, see their [Getting Started][5] page.
+1. [Create a Datadog account][1] if you haven't already.
+1. Find or create a [Datadog API key][2].
+1. Start up a Linux host or VM.
 
-```text
-vagrant init ubuntu/jammy64
-vagrant up
-vagrant ssh
-```
+## Create an application
 
-To install the Datadog Agent on a host, use the [one line install command][6] updated with your [Datadog API key][7]:
+To create an application to observe in Datadog:
 
-```shell
-DD_API_KEY=<DATADOG_API_KEY> DD_SITE="{{< region-param key="dd_site" >}}" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
-```
+1. On your Linux host or VM, create a new Python application named `hello.py`. For example, `nano hello.py`.
+1. Add the following code to `hello.py`:
 
-### Validation
+    {{< code-block lang="python" filename="hello.py" collapsible="true" disable_copy="false" >}}
+  from flask import Flask
+  import random
 
-Verify the Agent is running with the [status command][8]:
+  app = Flask(__name__)
+  
+  quotes = [
+      "Strive not to be a success, but rather to be of value. - Albert Einstein",
+      "Believe you can and you're halfway there. - Theodore Roosevelt",
+      "The future belongs to those who believe in the beauty of their dreams. - Eleanor Roosevelt"
+  ]
+  
+  @app.route('/')
+  def index():
+      quote = random.choice(quotes)+"\n"
+      return quote
+  
+  if __name__ == '__main__':
+      app.run(host='0.0.0.0', port=5050)
+  {{< /code-block >}}
 
-```shell
-sudo datadog-agent status
-```
+## Set up Datadog APM
 
-After a few minutes, verify the Agent is connected to your account by checking the [Infrastructure List][9] in Datadog.
+To set up Datadog APM without needing to modify your application's code or the deployment process, use Single Step APM Instrumentation:
 
-## Datadog APM
+<div class="alert alert-info"><strong>Note</strong>: <a href="https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/single-step-apm/">Single Step APM Instrumentation</a> is in beta. Alternatively, you can set up APM using <a href="https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd-libraries/">Datadog tracing libraries</a>.</div>
 
-### Follow the in-app documentation (recommended)
+1. Run the installation command:
 
-For the remaining steps, follow the [Quick start instructions][10] within the Datadog site for the best experience, including:
+   ```shell
+    DD_API_KEY=<YOUR_DD_API_KEY> DD_SITE="<YOUR_DD_SITE>" DD_APM_INSTRUMENTATION_ENABLED=host DD_ENV=<AGENT_ENV> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
```

Comment:
wondering if it makes sense to only install the `python` library as a part of this guide as thats what the example code is written in? It might speed up the install and less potential to break things. Unless part of the goal here is to expose users to things that we found that they did not explicitly intend to instrument?

@vidurkhannadatadog @jonbodner any thoughts?